egs2/voxceleb/spk1/run.sh

```diff
@@ -6,12 +6,14 @@ set -o pipefail
 
 spk_config=conf/train_RawNet3.yaml
```

Comment:
```suggestion
spk_config=train_rawnet3.yaml
```

Reply:
I will change this to `conf/train_rawnet3.yaml`